spec/models/card_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+describe Card do
+  it 'pass validation - Card' do
+    card = Card.new(original_text: 'schwarz', translated_text: 'black', review_date: Date.today)
```

Comment:
pls use let for variables, and change variable in context, something loke this:

pseudocode
```
describe Card do 
  let(:card) { FactoryBot.build(:card, card_params) } 
  let(:card_params) {{ original_text: 'schwarz', translated_text: 'black', review_date: Date.today }} 
  
  context 'with valid params do 
    it 'should pass validation' do 
       expect(card).to be_valid
    end
  end
  
  context 'with equal original and translate value' do 
   let(:card_params) {{original_text: 'schwarz', translated_text: 'SCHWARZ', review_date: Date.today}}
     it 'should not pass validation' do 
        expect(card).not_to be_valid
     end
  end
end
 ```